public/apps/configuration/app-router.tsx

```diff
@@ -64,7 +65,7 @@ export function AppRouter(props: AppDependencies) {
   return (
     <Router basename={props.params.appBasePath}>
       <EuiPage>
-        <Route path={`(${RoutesList.map((r) => r.href).join('|')})`} exact>
+        <Route path={`(${RoutesList.map((r) => r.href).join('|')}|/)`} exact>
```

Comment:
not easy to understand the arrow function, would you create function for readability

Reply:
Created a constant for it.